go-selinux/selinux_linux.go

```diff
@@ -606,17 +606,17 @@ func bitsetToStr(c *big.Int) string {
 	return str
 }
 
-func (l1 *level) equal(l2 *level) bool {
```

Comment:
I like `l1` and `l2` more in this case, since we compare them. A linter is not always right.